### DISCUSSION TITLE
A case that causes the GitHub Pages build to not start

### DISCUSSION THREAD

**Discussion by cotes2020** [community/community#23031] [Other Feature Feedback, Questions, & Ideas]
<p><strong>Notes</strong> : All the repos mentioned below are public.</p>
<p>I develop in jekyll-theme project <a href="https://github.com/cotes2020/chirpy" rel="nofollow noopener">chirpy</a> and its Travis-CI <a href="https://github.com/cotes2020/chirpy/blob/master/_scripts/travis/cibuild.sh" rel="nofollow noopener">build script</a> performs two tasks in sequence:</p>
<ol>
<li>Build a live demo site for chirpy and push the result to the repository <a href="https://github.com/cotes2020/chirpy-demo" rel="nofollow noopener">chirpy-demo</a>
</li>
<li>Build my blog site and push the site files to the repository <a href="https://github.com/cotes2020/cotes2020.github.io" rel="nofollow noopener">cotes2020.github.io</a>
</li>
</ol>
<p>After the completion of CI work, the GitHub Pages build of chirpy-demo will be triggered immediately but not for the repos <a href="http://cotes2020.github.io" rel="nofollow noopener">cotes2020.github.io</a>.</p>
<p>So I used <a href="https://github.com/cotes2020/chirpy/blob/9bd5f538b23189082741f60e23be2aa9976bec5a/_scripts/travis/cibuild.sh#L159" rel="nofollow noopener">the RESTful API</a> in Travis-CI build to trigger the Pages build of <a href="http://cotes202.github.io" rel="nofollow noopener">cotes202.github.io</a> and it works.<br>
To recover from the non-triggered build, I stopped sending the API trigger and here is the link to the non-triggered result:</p>
<p>In repos <a href="https://github.com/cotes2020/cotes2020.github.io" rel="nofollow noopener">cotes2020.github.io</a>,</p>
<ul>
<li>
<a href="https://github.com/cotes2020/cotes2020.github.io/commit/d06955cc5b8ab61ae75f5d51f59bf006592ab510" rel="nofollow noopener">The latest commit</a> is d06955c</li>
<li>
<a href="https://github.com/cotes2020/cotes2020.github.io/deployments" rel="nofollow noopener">The GitHub Pages deployment</a> use the old version commit d8d4e45, which proved that no Pages build was triggered</li>
</ul>
<p>Is there a restriction that does not allow two Pages sites to be built at the same time?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>You can run two Pages builds at the same time, yes. They may not be run concurrently, but they’ll both be queued.</p>
<p>From your description, however, it sounds like the behavior you’re describing is expected. Here’s what I understand from your post:</p>
<ol>
<li>You had a setup where you would push first to chirpy-demo, then to cotes2020 and only chirpy-demo Pages build would get triggered</li>
<li>Then you added something that specifically triggered a Pages build for cotes2020. This made both sites build</li>
<li>Then you took out the thing you added. Now only chirpy-demo builds automatically again</li>
</ol>
<p>If my understanding is correct, then this sounds like you don’t have cotes2020 configured to build correctly. Unfortunately, we don’t have access to your repository’s configuration details. You’ll need to contact private support to help debug why cotes2020 isn’t building automatically. You can contact them at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> or by email at <a href="mailto:support@github.com">support@github.com</a>.</p>
<p>I’m sorry I couldn’t be more help but please let me know if I’ve misunderstood.</p>

**Comment by cotes2020** [comment]
<p>Hi <a class="mention" href="/lee-dohm">@lee-dohm</a>，</p>
<p>Your understanding is absolutely correct and sorry for my bad English. I got in touch with <em>GitHub Developer Support</em>  a few days ago and they are dealing with this issue.</p>
<p>Thanks for your reply.</p>
